src/threading/thread.h

```diff
@@ -145,6 +137,14 @@ class Thread {
 
 	static void threadProc(Thread *thr);
 
+	/*
+	 * Immediately terminates the thread.
+	 * This should be used with extreme caution, as the thread will not have
+	 * any opportunity to release resources it may be holding (such as memory
+	 * or locks).
+	 */
```

Comment:
As this method is now private, I'm not sure this comment is still relevant. But a comment about why it as been made private could avoid any future mistake making it public again (mentioning BSD?).